build.gradle

```diff
@@ -117,6 +117,7 @@ subprojects {
     if (!name.toLowerCase().contains('test')) {
       options.errorprone {
         check('NullAway', CheckSeverity.ERROR)
+        check('InlineFormatString', CheckSeverity.OFF)
```

Comment:
Do we just have way too many instances of this?

Reply:
Not too many, but the problem was that, strangely, I was not able to make `@SuppressWarnings("InlineFormatString")` work [here](https://github.com/GoogleContainerTools/jib/blob/307364160f3189e3079f7f3820be673f5d42e7ed/jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/JibBuildRunner.java#L49-L62). Adding the annotation right before the constant fields or before the class didn't work. Not sure why.

Another nuisance was that it forces to remove the constants on the `*Step` classes (like [this one](https://github.com/GoogleContainerTools/jib/blob/307364160f3189e3079f7f3820be673f5d42e7ed/jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/BuildImageStep.java#L36)). The check complains only when the string is for a format string (e.g., containing `%s`), but for consistency, we'll have to remove non-format string constants as well.

Reply:
>but the problem was that, strangely, I was not able to make @SuppressWarnings("InlineFormatString") work here

I guess that was https://github.com/google/error-prone/issues/1650

Reply:
@PascalSchumacher ah, a bug in 2.4.0. Thanks for letting us know!

Reply:
I'll hold off merging whit until they release a new version that fixes the `@SuppressWarnings("InlineFormatString")` issue.